### DISCUSSION TITLE
zoom in nearest instead of interpolation

### DISCUSSION THREAD

**Discussion by 22anwar** [sylikc/jpegview#83] [Q&A]
is there any way to make the zooming in photos in nearest neighbor instead of interpolation?

**Comment by Bobbsterman** [comment]
There is, but it's somewhat limited. You need to assign a shortcut to IDM_TOGGLE_RESAMPLING_QUALITY. I think the default is F3?
However, it will only toggle it off for that image: it resets for every image, and every restart.
